cmd/apps/mongodb_app.go

```diff
@@ -100,7 +101,7 @@ func MakeInstallMongoDB() *cobra.Command {
 		}
 
 		err = helm3Upgrade(outputPath, "stable/mongodb",
-			namespace, "values.yaml", "",overrides)
+			namespace, "values.yaml", defaultVersion,overrides, wait)
```

Comment:
Please run `gofmt` on the code and send an updated patch. `gofmt -w -s ./filename.go`